sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3986,4 +3986,13 @@ class SQLConf extends Serializable with Logging {
   def isModifiable(key: String): Boolean = {
     sqlConfEntries.containsKey(key) && !staticConfKeys.contains(key)
   }
+
+  /**
+   * Return all non-static configuration
+   */
+  def getAllNonStaticConfs(): Map[String, String] = {
```

Comment:
When do we need to declare this additional function? This PR shows the modified configs by comparing the value between `sessionState.conf` and `globalConfigs`. For the static configs, the values are the same already, aren't they?

Reply:
Removed it since it's redundant.